pkg/config/generator.go

```diff
@@ -70,6 +73,39 @@ func NewMultusConfig(pluginName string, cniVersion string, kubeconfig string, co
 // Generate generates the multus configuration from whatever state is currently
 // held
 func (mc *MultusConf) Generate() (string, error) {
+	versionFmt := "delegate cni version is %s while top level cni version is %s"
+	v040, _ := semver.Make("0.4.0")
+	v031, _ := semver.Make("0.3.1")
+	topLevel, err := semver.Make(mc.CNIVersion)
```

Comment:
please name this `multusCNIVersion` or something alike. 

I don't expect a version number to be in a variable named `topLevel` .